package.json

```diff
@@ -36,6 +36,7 @@
 		"escape-html": "^1.0.3",
 		"hark": "^1.2.3",
 		"lodash": "^4.17.20",
+		"lru-cache": "^6.0.0",
```

Comment:
This is reaching a level where i think it stops being useful/worth it

Reply:
It is only ~16 KiB unpacked including its single dependency, so I do not think it is so bad :-) If I had written my own with just the needed features it would have been (I hope...) smaller, but this was faster and easier :shrug:

Reply:
well its not about the bytes or whatever, its about the complexity off all this. 3rdparty lib, caching, loading images and blurring them with JS, …